coffee-warehouse/src/components/GridComponents/BudgetComponent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <span v-if="dataItem.budget < 0" class="red text-bold"

```

Comment:
instead of changing all the element lets just change the class depending on the budget prop

Reply:
Fixed with commit https://github.com/telerik/kendo-vue/pull/164/commits/78129372ac4fbe53afb4c2e3814f9ab9ee66f2f5